package.json

```diff
@@ -35,7 +35,8 @@
       "temperature": "bricklets/Temperature.js",
       "digiin": "bricklets/IndDigital4In.js",
       "analogout": "bricklets/AnalogOut.js",
-      "indanalogout": "bricklets/IndAnalogOut.js"
+      "indanalogout": "bricklets/indAnalogOut.js",
```

Comment:
It shouldn't make it hang, it should just complain it can't find the node but maybe combined with the next change. Fix both and try, if it still hangs open a new issue on this repo and include the output of node red -v